docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -1,27 +1,22 @@
 [id="migrating-databases-to-mariadb-instances_{context}"]
 
-//kgilliga: Find out if the steps in the Variables and pre-checks sections can go in the main procedure or if they have to be done before.
-
 = Migrating databases to MariaDB instances
 
-This document describes how to move the databases from the original
-{rhos_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
+Migrate your databases from the original {rhos_prev_long} ({OpenStackShort}) deployment to the MariaDB instances in the {OpenShift} cluster.
 
-[NOTE]
-This example scenario describes a simple single-cell setup. Real
-multi-stack topology recommended for production use results in different
-cells DBs layout, and should be using different naming schemes (not covered
-here this time).
+//[NOTE]
+//This example scenario describes a simple single-cell setup. Real
+//multi-stack topology recommended for production use results in different
+//cells DBs layout, and should be using different naming schemes (not covered
+//here this time). kgilliga: I hid the same note in the Compute adoption procedure. Will likely reinstate this after multi-cell is released.
 
 .Prerequisites
 
-* Make sure the previous Adoption steps have been performed successfully.
- ** The `OpenStackControlPlane` resource must be already created.
```

Comment:
Is “* The `OpenStackControlPlane` resource must be already created,” a prerequisite for the entire adoption guide, not just this procedure?

Reply:
Except the initial step where the `OpenStackControlPlane` is created, yes.

Reply:
I think we should create osp secrets before?

Reply:
I think that this is part of a broader conversation of what prerequisites are required before the entire adoption procedure is started, vs, what prerequisites are required for this specific procedure.

I don't think we need to delay merging this PR for that conversation to be resolved. We can revisit this procedure later if needed.